.github/workflows/ci.yml

```diff
@@ -64,7 +64,25 @@ jobs:
     - uses: Swatinem/rust-cache@v2
     - uses: taiki-e/install-action@cargo-hack
     - name: Default features
-      run: cargo hack check --feature-powerset --locked --rust-version --ignore-private --workspace --lib --bins
+      run: cargo hack check --feature-powerset --locked --rust-version --ignore-private --workspace --all-targets
```

Comment:
Going from `--lib --bins` to `--all-targets` causes this job to fail as we have a `dev-dependency` on `divan` which appears to have an MSRV of ~`1.70.0` while our MSRV is `1.65.0`.